applications/Interstellar-Network.md

```diff
@@ -0,0 +1,575 @@
+
+If this is an application for a follow-up grant (the continuation of an earlier, successful W3F grant), please provide name and/or pull request of said grant on the first line of this section.
+
+### Overview
+
+- An indication of how your project relates to / integrates into Substrate / Polkadot / Kusama.
+- An indication of why your team is interested in creating this project.
+
+![Copy of Interstellar-Black-Text](https://user-images.githubusercontent.com/4605611/141333053-3f607ffe-1714-4512-b628-33274d0d0464.png)
+
+
+ **Interstellar Wallet Phase 1: Substrate OCW Garbled Circuit Factory GCF and Trusted Transaction Validation protocol pallet demo (as a GCF use case)** 
+ 
+#### Why Interstellar wallet?
+
+**User experience** and **Security** are still the main **pain points** of non-custodial wallet. We think that they slow down blockchain and DeFi adoption.
+
+- **Wallets are not easy for newcomers and even for some experienced users:**
+
+The Public Private key concept is complex for non-technical people and can be easily misused. The process of wallet creation can be a bit confusing and misleading, and the average user still do not understand clearly what information, passphrase, private key, etc. need to be protected and how to store it safely.
+
+- **Only Hardware Wallets (like Ledger, Trezor, Ngrave, etc.) are secures:**
+
+Because blockchains themselves and centralized exchanges are getting more secures, bad actors now focus their attacks on Individual wallets and exchange accounts. 
+
+Banking trojans are now also targeting cryptocurrencies's owners. Dapp and wallets are more and more exposed on mobiles and in browsers as those trojans are getting more sophisticated and pervasive.
+**“Android banking malware, which rose by an incredible 158.7 per cent in Q1, saw a continued increase of 49 per cent”**- Source ESET quarterly threat report 2021. Zero day that enables bad actors, through “exploit chain”, to get the control on mobile operating system and browsers have never been higher.[2021 has broken the record for zero-day hacking attacks]( https://www-technologyreview-com.cdn.ampproject.org/c/s/www.technologyreview.com/2021/09/23/1036140/2021-record-zero-day-hacks-reasons/amp/)
+
+Given the growth dynamic of the malware/banking trojan threat, the blockchain ecosystem need now a protection against malware on mobiles wallet and Dapps.
+We think that Hardware security, Trusted Execution Environnement TEE and especially Trusted User Interface TUI can provide a level of hardware security comparable to hardware wallet. Mobile UI security is crucial on this respect to address current and growing threats (including banking trojan with their overlay capabilities). They can easily build fake UI that mislead user to trigger tampered transaction. From a pure security standpoints mobile banking application (DSP2 compliant) are now more robust than mobile wallet. As a consequence, threat actors focus now on wallets.
```

Comment:
"a level of hardware security comparable to hardware wallet" -> "a level of security comparable to hardware wallet" ?

Reply:
"They can easily build fake UI that mislead user to trigger tampered transaction." -> "They can easily build fake UIs that mislead users to trigger a tampered transaction." ou plutot tout au singulier?